### DISCUSSION TITLE
SF#1381, cannot search for vendor using entity control code not clear that this is not supposed to work

### DISCUSSION THREAD

**Discussion by einhverfr** [ledgersmb/LedgerSMB#5825] [Design]
Related to #3994:

https://sourceforge.net/p/ledger-smb/bugs/1381/

Per request from ehu, lodging bug:

An attempt to create an AP transaction from vendor's control code (
![ap-new-tx](https://user-images.githubusercontent.com/2326559/121077831-6669b880-c7d8-11eb-9e99-caebdf38cf16.PNG)
) produces error on clicking 'save':

```
Error!

'Vendor not on file! at bin/arap.pl line 148.
'

dbversion: 1.4.12, company: lsmb14
```

However, entering the numeric credit account ID (per contact › credit accounts) worked.

**Comment by ehuelsmann** [comment]
Nothing changed here, but can't change it in any stable releases. Putting in 1.6; expecting to change customer selection in 1.7 (or so) entirely  (when we have services), so, probably best to handle at that time.
